CONTRIBUTING.md

```diff
@@ -1,135 +1,96 @@
-# Přidej se k vývoji Movapp na Apple zařízení
+# Before you continue reading this content please read first our [Movapp CONTRIBUTING.MD](https://github.com/cesko-digital/movapp/blob/ed0d95a196568e9055c5f4ca347647cfed790aae/CONTRIBUTING.md)
```

Comment:
I think this doesn look very nice.

```md

# How to contribute to iOS version of Movapp

As first step, read our general guidelines for all [movapp.cz](https://movapp.cz) projects by reading this [CONTRIBUTION.md file](https://github.com/cesko-digital/movapp/blob/main/CONTRIBUTING.md)

```

Link should point to a main version right?